CMakeLists.txt

```diff
@@ -811,11 +811,11 @@ MESSAGE(
 MESSAGE(
 "Developer options\n"
 "-----------------------------------------\n"
-"* Debug FP exceptions         : ${STATUS_DEBUG_FPE}\n"
-"* Debug AddressSanitizer      : ${STATUS_DEBUG_ASAN}\n"
-"* Debug ThreadSanitizer       : ${STATUS_DEBUG_TSAN}\n"
-"* Debug MemorySanitizer       : ${STATUS_DEBUG_MSAN}\n"
-"* Debug UBSanitizer           : ${STATUS_DEBUG_UBSAN}\n"
+"* Debug using FP exceptions         : ${STATUS_DEBUG_FPE}\n"
```

Comment:
I think this one's fine to leave as is, but it doesn't matter too much - I think the floating-point exceptions can be classified as both a tool (in that they don't normally occur, and are specifically turned on to find potentially problematic calculations), and the thing being debugged (in that you look at the stack trace of them as you would any other exception).

Reply:
I changed it for consistency. I didn't like how it wasn't alligned with the other status messages. If the preference is towards leaving it be, I don't have a problem with that.